src/common/procedure/src/local/runner.rs

```diff
@@ -104,6 +105,7 @@ impl Drop for ProcedureGuard {
 
 // TODO(yingwen): Support cancellation.
```

Comment:
You can remove this todo now.
```suggestion
// TODO(yingwen): Support cancellation.
```